internal/visualization/console/feature_map_test.go

```diff
@@ -0,0 +1,27 @@
+package console
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("#FeaturesForOCP", func() {
+	It("should return an empty set when there is an empty version ", func() {
+		Expect(FeaturesForOCP("")).To(Equal([]string{}))
+	})
+	It("should return the default set when there is no version match ", func() {
```

Comment:
It's unclear what the version gets matched against.

Reply:
is your point it should be called something like "default" or something more verbose then "unamatched"?